drivers/misc/devmux/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2023, Meta
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+zephyr_syscall_header(
+  ${ZEPHYR_BASE}/include/zephyr/drivers/misc/devmux/devmux.h
+)
```

Comment:
~is this strictly required? (sorry, was not aware of this)~

edit: I guess it's needed because there's no `__subsystem` tag.

Reply:
It is needed otherwise the system calls aren't generated. I don't know the exact details of the machinery but we added these when @dcpleung made some changes a while back to only generate the minimal number of syscalls